docs/secure-services.md

```diff
@@ -117,3 +118,38 @@ make build image
 
 Once built locally, be sure to push it to the selected registry and *use the
 image in the* [controller deployment](https://github.com/EOSC-synergy/ingress-nginx/blob/custom/controller-v0.41.2/deploy/static/provider/baremetal/deploy.yaml#L332).
+
+
+### Testing the API
+Get the refresh and access token from [EGI Check-in AAI dev](https://aai-dev.egi.eu/fedcloud/) using Fedcloud web interface. After the OIDC validation in web interface, the access token can be generated using the provided command that should appear there with the following syntax:
+```
+curl -X POST -u '<client id>':'<client secret>' -d 'client_id=<client id>&client_secret=<client secret>&grant_type=refresh_token&refresh_token=<refresh token>&scope=openid%20email%20profile' 'https://aai-dev.egi.eu/oidc/token' | python -m json.tool;
```

Comment:
Here you need the `<refresh-token>`, but there is no hint about how to get it. If it is related to the outcome of the former "OIDC validation in the web interface" statement, we should indicate how to tackle this last action

Reply:
Yes, you're right is not explicit...
When using the web interface it lists multiple properties as the result and one of them is the <refresh-token>. They also provide the complete oneliner with all fields already fulfilled.
@orviz Can you test the suggested procedure and see if any other detail is missing?

Reply:
Maybe it would be good to add a screenshot:

![image](https://user-images.githubusercontent.com/1225231/102877645-5d15ab00-4447-11eb-90fa-0497879b9a7c.png)

BTW didn't work with Chromium, only with Firefox. If you can reproduce this, it should be helpful to document the issue